.npmrc

```diff
@@ -1 +1,2 @@
 git-checks=false
+public-hoist-pattern[]=@img*
```

Comment:
Why is this needed?

Reply:
The fixtures CI checks fail without it, something to do with how sharp imports its native dependencies?

Reply:
Actually, I needed to mark `sharp` as unbundled - works if I do that